src/include/daos_srv/daos_engine.h

```diff
@@ -516,6 +516,7 @@ int dss_ult_execute(int (*func)(void *), void *arg, void (*user_cb)(void *),
 		    void *cb_args, int xs_type, int tgt_id, size_t stack_size);
 int dss_ult_create_all(void (*func)(void *), void *arg, bool main);
 int __attribute__((weak)) dss_offload_exec(int (*func)(void *), void *arg);
+int __attribute__((weak)) dss_load_exec(void (*func)(void *), void *arg);
```

Comment:
It's not specific for loading, maybe use a common name like dss_main_exec() or dss_vos_exec()?

Reply:
dss_main_exec() looks better.